common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -910,6 +910,11 @@ div.problem .action {
     }
   }
 
+  .reset-error{
+    color: orangered;
+    display: none;
+  }
```

Comment:
@mehrozezahid Ideally we should use [copy-error](https://github.com/edx/edx-platform/blob/master/lms/static/sass/course/instructor/_instructor_2.scss#L118) class but this is only available in instructor dashboard so you can move it into [_developer.scss](https://github.com/edx/edx-platform/blob/master/lms/static/sass/_developer.scss), then it should be available at all places.